compiler/crates/relay-bin/src/main.rs

```diff
@@ -205,6 +209,10 @@ async fn handle_compiler_command(command: CompileCommand) -> Result<(), Error> {
 
     let mut config = get_config(command.config)?;
 
+    if let Some(project_to_retain) = command.project {
+        config.retain_project(project_to_retain);
+    }
+
```

Comment:
It looks like we can just set the project as `enabled` in the config. Here's what we do in the internal version:

```rust
/// Update Config if the `project` flag is set
fn set_project_flag(config: &mut Config, project: Vec<String>) {
    if project.is_empty() {
        return;
    }
    for project_config in config.projects.values_mut() {
        project_config.enabled = false;
    }
    for selected_project in project {
        let selected_project = selected_project.intern();
        if let Some(project_config) = config.projects.get_mut(&selected_project) {
            project_config.enabled = true;
        } else {
            println!(
                "Project `{}` not found, available projects: {}.",
                selected_project,
                config
                    .projects
                    .keys()
                    .map(|name| name.lookup())
                    .collect::<Vec<_>>()
                    .join(", ")
            );
            std::process::exit(1);
        }
    }
}
```